core/templates/dev/head/pages/skill_player/ConceptCardDirective.js

```diff
@@ -0,0 +1,50 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for the Concept Card.
+ */
+oppia.directive('conceptCard', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        returnToExploration: '&',
+        getSkillId: '&skillId'
+      },
+      templateUrl: UrlInterpolationService.getDirectiveTemplateUrl(
+        '/pages/skill_player/concept_card_directive.html'),
+      controller: [
+        '$scope', 'ConceptCardBackendApiService',
+        function(
+            $scope, ConceptCardBackendApiService) {
+          ConceptCardBackendApiService.fetchConceptCard(
+            $scope.getSkillId()).then(function(conceptCard) {
+            $scope.skillExplanation = conceptCard.explanation;
```

Comment:
This needs a domain object. Can't refer directly to backend fields like this.

Also, just wondering -- do we need Skills anywhere else in the learner view? If so then perhaps we should be using a Skill data fetcher instead and then we can probably reuse domain object classes in the frontend.

Reply:
The only field not used in skills is misconceptions which is only applicable for questions. I think skills might be used for practice questions after showing the concept card to populate the misconception feedback.
Also, there is a ConceptCardObjectFactory, shall I use that itself?

Reply:
Yep, go ahead and use that domain object. The question of whether we should have one concept (Skill) or two concepts (Skill and ConceptCard) is your call -- just add a note here saying what you decide, for posterity.

Reply:
Ok, since the user facing part of the skill is the concept card, I am going with a concept card object in skill_player for now. The other field in the skill is misconceptions which is only used by a creator of a question to tag answer groups with a skill, in which case, a concept card is not required. So, misconception and concept card can be treated as independent objects.

Reply:
Done.